netstandard/src/netstandard.csproj

```diff
@@ -40,26 +40,36 @@
   </ItemGroup>
   <ItemGroup Condition="'$(TargetGroup)'=='xamarin.ios'">
```

Comment:
Yeah. In fact, wouldn't it be better if we just do `<ProjectReference Include="..\..\platforms\$(TargetGroup)\*.csproj" />` and get rid of the conditions altogether?